addons/full-import/import.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -o nounset -o pipefail -o errexit
+
+source /usr/local/pf/addons/full-import/helpers.functions
+source /usr/local/pf/addons/full-import/database.functions
+source /usr/local/pf/addons/full-import/configuration.functions
+
+dump_path="$1"
+
+if ! [ -f "$dump_path" ]; then
+  echo "Please specify a valid archive to extract"
+  echo "Usage: import.sh /path/to/export.tgz"
+  exit 1
+fi
+
+extract_dir=`mktemp -d`
+
+cp -a $dump_path $extract_dir/export.tgz
+check_code $?
+
+pushd $extract_dir/
+check_code $?
+
+main_splitter
+
+echo "Extracting archive..."
+tar -xf export.tgz
+check_code $?
+
+echo "Found the following content in the archive:"
+ls -l
```

Comment:
We should exclude "export.tgz", otherwise, `ls -l` displays `export.tgz` and you think  `export.tgz` is part of the archive.